downstream/modules/platform/proc-verify-connectivity.adoc

```diff
@@ -0,0 +1,9 @@
+[id="proc-verify-connectivity_{context}"]
+
+= Verifying Connectivity
```

Comment:
This seems like it could be a "next steps" or verification step in a procedure, rather than a separate module.